scripts/release_scripts/update_changelog_and_credits_test.py

```diff
@@ -33,31 +33,55 @@
 from scripts import common
 from scripts.release_scripts import update_changelog_and_credits
 
-import github  # isort:skip pylint: disable=wrong-import-position
+from typing import List, Union
+from typing_extensions import Final
```

Comment:
There should be no empty line below this one I think.

Reply:
Removed!